modules/twinkleunlink.js

```diff
@@ -171,7 +171,7 @@ Twinkle.unlink.callbacks = {
 					if ($(xmlDoc).find('query-continue').length) {
 						apiobj.params.form.append({
 							type: 'div',
-							label: 'First ' + list.length.toString() + ' file usages shown.'
+							label: 'First ' + mw.language.convertNumber(list.length.toString()) + ' file usages shown.'
```

Comment:
Doesn't this negate the need for `toString()`?  AFAICT `mw.language.convertNumber` returns a string unless we pass a truthy second parameter.  Below as well.